contracts/nft/NFTFacet.sol

```diff
@@ -120,7 +120,13 @@ contract NFTFacet is RolesMods {
             msg.sender == address(TELLER_NFT_V2),
             "Teller: unaccepted 1155"
         );
-        __stakeNFTV2(from, id, value);
+
```

Comment:
we can add a `bool` to check if it's bridging it or not which will help with the following logic gate